README.adoc

```diff
@@ -63,20 +63,89 @@ Actually things aren't that bad
 --
 ----
 
-=== Background colors
-
 In previous snippet we are creating a slide titled Slide One with bullets and another one titled Slide Two with centered text (reveal.js' default behavior) with https://github.com/hakimel/reveal.js#speaker-notes[speaker notes].
 
+
+=== Slides without titles
+
+There are a few ways to have no titles on slides.
+
+* Setting your title to `!`
+* Adding the `notitle` option to your slide
+* Adding the `conceal` option to your slide
+
+Here is an example of the three techniques in action:
+
+----
+include::test/concealed-slide-titles.adoc[]
 ----
-[data-background="yellow"]
+
+NOTE: `conceal` and `notitle` have the advantage that the slide still has an id so it can be linked to.
+
+
+=== Background colors
+
+----
+[background-color="yellow"]
 == Slide Three
 
 Is very yellow
 ----
 
-Slide Three applies the attribute http://lab.hakim.se/reveal-js/#/14[data-background] to the `reveal.js` <section> tag.
+Slide Three applies the attribute
+https://github.com/hakimel/reveal.js/#slide-backgrounds[data-background-color] to the `reveal.js` <section> tag.
 Anything accepted by CSS color formats works.
-Here is the https://github.com/hakimel/reveal.js#slide-backgrounds[upstream documentation].
+
+
+=== Background images
+
+----
+[%notitle]
+== Grand Announcement
+
+image::cover.jpg[background, size=cover]
+----
+
+This will put `cover.jpg` as the slide's background image.
+It sets `reveal.js'` `data-background-image` attribute.
+The `size` attribute is also supported.
+See the https://github.com/hakimel/reveal.js/#image-backgrounds[
+reveal.js' documentation] for details.
+
+NOTE: Background images file names are now relative to the `:imagedir:`
+attribute if set.
+
+NOTE: The `canvas` keyword can be used instead of `background` for the same
+effect.
```

Comment:
I prefer to use the sentence-per-line formatting instead of hard wrapping at a fixed column. Would you agree to use that style?